npm-package/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "npm-catalogue",
+  "version": "1.0.0",
+  "private": true,
+  "workspaces": [
+    "packages/*",
+    "plugins/*"
```

Comment:
Let's remove this `plugins` workspace, both from here and the [pnpm-workspace.yaml](https://github.com/sliit-foss/nodejs-templates/pull/1/files#diff-5f647fbcca6b22bfe7ced86593890e1f26e092b32b76cf2c811457748f959ad1) file. Let's remove the babel plugin and the eslint plugins included as well and add a simple package like one which contains a few calculator functions to the packages workspace to keep things simple

Reply:
How about "lodash"?

Reply:
Didn't quite get this, do you mean a dummy package named lodash ?

Reply:
Added an example package with a simple calc function.